### ISSUE TITLE
Add Mavenhive as Associate Sponsor

### ISSUE THREAD

**Issue by smtalimett** [rubyconfindia/2019#76]
Logo you have?

**Comment by prathamesh-sonpatki** [comment]
Will take logo from devopsdays site.

./prathamesh

 GITHUB <https://github.com/prathamesh-sonpatki> | | BigBinary
<https://BigBinary.com>

**Comment by prathamesh-sonpatki** [comment]
@IndianGuru Do we have writeup for Mavenhive?

**Comment by smtalimett** [comment]
No. Have asked them to give.

**Comment by smtalimett** [comment]
MavenHive is a Bangalore based tech consulting firm specializing in end-to-end product development and training. Over the last 5 years, we have helped several startups and organizations from across the world realize their product vision and achieve major milestones in their journey. We have built a team of talented people from diverse backgrounds who have contributed their experience and skills to deliver high quality solutions for our clients. We are always looking for great developers to work with, to strengthen our capabilities and influence our work culture - www.mavenhive.in
